balancer/grpclb/grpclb_test.go

```diff
@@ -1102,10 +1103,11 @@ func (s) TestGRPCLB_PickFirst(t *testing.T) {
 		grpc.WithTransportCredentials(&serverNameCheckCreds{}),
 		grpc.WithContextDialer(fakeNameDialer),
 	}
-	cc, err := grpc.Dial(r.Scheme()+":///"+beServerName, dopts...)
+	cc, err := grpc.NewClient(r.Scheme()+":///"+beServerName, dopts...)
 	if err != nil {
-		t.Fatalf("Failed to dial to the backend %v", err)
+		t.Fatalf("Failed to create a newclient: %v", err)
 	}
+	cc.Connect()
```

Comment:
nit: We can replace the `r.UpdateState` with `r.InitialState` at line 1119 below instead of using `cc.Connect` here.

Reply:
You will also need to replace the call to `r.CC.ParseServiceConfig()` below at line 1115 with `internal.ParseServiceConfig.(func(string) *serviceconfig.ParseResult)()`. There are other usages of `internal.ParseServiceConfig` in this file that you can refer to.

Reply:
Fixed this.